sql/core/src/test/resources/sql-tests/inputs/string-functions.sql

```diff
@@ -48,4 +48,8 @@ SELECT trim(LEADING 'xyz' FROM 'zzzytestxyz');
 SELECT trim(LEADING 'xy' FROM 'xyxXxyLAST WORD');
 SELECT trim(TRAILING 'xyz' FROM 'testxxzx');
 SELECT trim(TRAILING 'xyz' FROM 'xyztestxxzx');
-SELECT trim(TRAILING 'xy' FROM 'TURNERyxXxy');
\ No newline at end of file
+SELECT trim(TRAILING 'xy' FROM 'TURNERyxXxy');
+
+-- Check lpad/rpad throw an exception for invalid length input
```

Comment:
it only throws exception under ansi mode. how about
```
-- Check lpad/rpad with invalid length parameter
```

Reply:
Ur, my bad. It looks better. I'll update it.